configs/mailer.ts

```diff
@@ -0,0 +1,19 @@
+import nodemailer from 'nodemailer';
+import SMTPTransport from 'nodemailer/lib/smtp-transport';
+
+const host = process.env.EMAIL_SERVER_HOST;
+const email = process.env.EMAIL_SERVER_USER ?? '';
+const pass = process.env.EMAIL_SERVER_PASSWORD ?? '';
+const port = Number(process.env.EMAIL_SERVER_PORT) ?? 0;
+const transport: SMTPTransport.Options = {
+  host,
+  port,
+  secure: true,
+  // service: 'gmail',
+  auth: {
+    type: 'LOGIN',
+    user: email,
+    pass,
+  },
+};
+export const transporter = nodemailer.createTransport(transport);
```

Comment:
This really limited. Let's create at least 1 default transporter wrapper for SendGrid, use the following documentation:

- how to create our own wrapper: https://nodemailer.com/transports/sendmail/
- inside use use official SendGrid SDK so we don't have to deal with detail implementation: https://github.com/sendgrid/sendgrid-nodejs
- example: https://github.com/orliesaurus/nodemailer-mailgun-transport/blob/master/src/index.js